hebiros_advanced_examples/src/omnibase/example_omnibase_free_roam.cpp

```diff
@@ -110,16 +110,16 @@ int main(int argc, char ** argv) {
   // Construct the group using 3 known modules, connected to the network
   AddGroupFromNamesSrv add_group_srv;
   add_group_srv.request.group_name = group_name;
-  add_group_srv.request.names = {"_Wheel1", "_Wheel2", "_Wheel3"};
+  add_group_srv.request.names = {"Wheel1", "Wheel2", "Wheel3"};
```

Comment:
Note for future PR -- this should probably be a parameter that is checked when starting the node.

We've got a lot of examples in the `hebi_cpp_api_examples` package that function like this; at some point breaking that function call into a utility helper function would be great.